Cargo.toml

```diff
@@ -12,6 +12,7 @@ repository="https://github.com/matrix-org/seshat/"
 default = ["encryption"]
 encryption = ["rusqlite/sqlcipher", "aes", "crypto-mac", "hmac", "sha2",
               "hkdf", "pbkdf2", "rand", "zeroize", "byteorder"]
+encryption-bundled = ["encryption", "rusqlite/bundled-sqlcipher-vendored-openssl"]
```

Comment:
Oh, is this a new `rusqlite` thing? I don't remember this being a thing. Don't we need to bump the version?

Let's call this `bundled-sqlcipher`, the Rust Crypto stuff obviously doesn't get bundled.

Reply:
Both done :)